kubernetes/windows/Dockerfile

```diff
@@ -62,16 +62,22 @@ COPY ./amalogswindows/installer/scripts/*.rb /opt/amalogswindows/scripts/ruby/
 # copy out_oms.so file
 COPY ./amalogswindows/out_oms.so /opt/amalogswindows/out_oms.so
 
+# copy containerinventory.so and perf.so file
+COPY ./amalogswindows/containerinventory.so /opt/fluent-bit/bin/containerinventory.so
+COPY ./amalogswindows/perf.so /opt/fluent-bit/bin/perf.so

```

Comment:
containerinventory.so and /perf.so need to be code signed similar to out_oms.so

Reply:
![image](https://github.com/microsoft/Docker-Provider/assets/25573844/d0f14f6f-6e36-4780-b149-bdf8c8973654)

Reply:
Added the change